plugins/network-elements/nuage-vsp/pom.xml

```diff
@@ -28,6 +28,13 @@
     <version>4.6.0-SNAPSHOT</version>
     <relativePath>../../pom.xml</relativePath>
   </parent>
+  <dependencies>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>${cs.lang3.version}</version>
+    </dependency>
+  </dependencies>
```

Comment:
@nlivens CloudStack already has a Pair class, ``com.cloud.utils.Pair``.  Why are we bringing another dependency for functionality that is already available?